src/main.ts

```diff
@@ -100,9 +100,12 @@ async function getApps(): Promise<App[]> {
 
   return (responseJson.items as App[]).filter(app => {
     return (
-      app.spec.source.repoURL.includes(
-        `${github.context.repo.owner}/${github.context.repo.repo}`
-      ) && (app.spec.source.targetRevision === 'master' || app.spec.source.targetRevision === 'main' || app.spec.source.targetRevision === 'HEAD')
+      app.spec.source.repoURL.includes(`${github.context.repo.owner}/${github.context.repo.repo}`)
+      && (
+          app.spec.source.targetRevision === 'master'
+          || app.spec.source.targetRevision === 'main'
+          || app.spec.source.targetRevision === 'HEAD')
+      && (!EXCLUDE_PATHS.includes(app.spec.source.path))
```

Comment:
Extra `(` here isn't needed:
```suggestion
      && !EXCLUDE_PATHS.includes(app.spec.source.path)
```